packages/d3fc-axis/README.md

```diff
@@ -128,3 +122,40 @@ const axis = fc.axisBottom(scale)
 ```
 
 <img src="screenshots/color.png"/>
+
+### Axis
+
+![](screenshots/axis.png)
+
+<a name="axisBottom" href="#axisBottom">#</a> fc.**axisBottom**()  
+<a name="axisTop" href="#axisTop">#</a> fc.**axisTop**()  
+<a name="axisLeft" href="#axisLeft">#</a> fc.**axisLeft**()  
+<a name="axisRight" href="#axisRight">#</a> fc.**axisRight**()  
+
+A drop-in replacement for D3 axis, see the [D3 documentation for API reference](https://github.com/d3/d3-axis#api-reference).
+
+### Ordinal Axis
+
+![](screenshots/ordinal.png)
+
+<a name="axisOrdinalBottom" href="#axisOrdinalBottom">#</a> fc.**axisOrdinalBottom**()  
+<a name="axisOrdinalTop" href="#axisOrdinalTop">#</a> fc.**axisOrdinalTop**()  
+<a name="axisOrdinalLeft" href="#axisOrdinalLeft">#</a> fc.**axisOrdinalLeft**()  
+<a name="axisOrdinalRight" href="#axisOrdinalRight">#</a> fc.**axisOrdinalRight**()  
+
+A drop-in replacement for D3 axis, see the [D3 documentation for API reference](https://github.com/d3/d3-axis#api-reference).
+
+### Axis Label Offset
```

Comment:
@DevAndyLee can you please add similar docs for the `axisLabelRotate` adapter?

Reply:
Done